var/spack/repos/builtin/packages/intltool/package.py

```diff
@@ -36,7 +36,7 @@ class Intltool(AutotoolsPackage):
     version('0.51.0', '12e517cac2b57a0121cda351570f1e63')
 
     # requires XML::Parser perl module
-    # depends_on('perl@5.8.1:', type='build')
+    depends_on('perl-xml-parser', type=('build', 'run'))
```

Comment:
I' ve checked, it seems to me it works: 
I think it works i because recently run deps of build deps have been added